include/input_mappings.hpp

```diff
@@ -15,8 +18,79 @@ struct InputMappings {
 	}
 
 	void setMapping(Scancode scancode, u32 key) { container[scancode] = key; }
+
+	void serialize(const std::filesystem::path& path) const {
```

Comment:
Needs a frontend argument string which will dictate which frontend to save mappings for, trying to interpret QtKey values as SDL scancodes or vice versa will likely explode